wiki/Placeholders.md

```diff
@@ -522,6 +523,33 @@ A majority of these Expansions are maintained by the PlaceholderAPI team and can
 
 ----
 
+- ### **ChangeOutput**
+  > /papi ecloud download changeitem
+
+  Alows you to change the output based on what other placeholders return
+  
+  ```
+  %changeoutput_<options>_<input>_<matcher>_<output-if-matched>_<output-if-not-matched>%
+  ```
+  
+  * \<options>
```

Comment:
Just a suggestion, but you could use inline code here, no?
Like `<options>`

But again just a suggestion

Reply:
I Just copied the style Kaliber had on his wiki. I don't really mind any style. As long as its good to be on this wiki

Reply:
alr. does this look good?